packages/venia-concept/src/components/CreateAccount/validators.js

```diff
@@ -22,21 +22,21 @@ const isPasswordComplexEnough = (str = '') => {
 
 export const validators = new Map()
     .set('confirm', (value, values) => {
-        return value !== values.password ? 'Passwords must match.' : null;
+        return value !== values.password ? 'Passwords must match.' : undefined;
     })
     .set('email', value => {
         const trimmed = (value || '').trim();
 
         if (!trimmed) return 'An email address is required.';
         if (!trimmed.includes('@')) return 'A valid email address is required.';
 
-        return null;
+        return undefined;
```

Comment:
Minor quibble: `return undefined` is [redundant](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return#Syntax).
```suggestion
        return;
```